app/src/main/AndroidManifest.xml

```diff
@@ -125,6 +125,12 @@
     <activity android:name=".testing.HtmlParserTestActivity" />
     <activity android:name=".testing.HomeTestActivity" />
     <activity android:name=".testing.InputInteractionViewTestActivity" />
+    <activity android:name=".testing.ImageRegionSelectionTestActivity" >
+      <intent-filter>
+        <action android:name="android.intent.action.MAIN" />
+        <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Why this change? I don't think we actually need any of the test activities to be launchable since Espresso launches them for us.

Reply:
yes that was done for some testing will revert this